kernel/arch/dreamcast/include/dc/video.h

```diff
@@ -250,19 +246,43 @@ int vid_check_cable(void);
 
     \param  base            The offset within VRAM to set the base to.
 */
-void vid_set_start(uint32 base);
+void vid_set_start(uint32_t base);
+
+/** \brief   Get the VRAM base of a framebuffer.
+    \ingroup video_fb
+
+    This function gets the position of the specified framebuffer within VRAM.
+    Any invalid fb value will be treated as the current framebuffer.
+
+    \param  fb            The number of the framebuffer or -1 for current.
+*/
+uint32_t vid_get_start(int32_t fb);
```

Comment:
```suggestion
uintptr_t vid_get_start(ssize_t fb);
```

Reply:
This would mean also rewriting each instance of the fb index to be size_t too though, as well as fb_curr? My understanding was that size_t was used for things that could be a possible output of sizeof, which *can* be used as an index (for example buf[size_t var] after doing a sizeof(buf)) but that it's not for just any 'ol index value. As is, there's no array that you can plug that fb value into.